cutevariant/gui/infovariantwidget.py

```diff
@@ -34,12 +38,14 @@ def set_variant(self, variant):
         self.view.clear()
 
         # print(variant)
-        for key, val in variant.items():
+        # Filter None values
+        g = ((key, val) for key, val in variant.items() if val)
+
+        for key, val in g:
             item = QTreeWidgetItem()
             item.setText(0, str(key))
             item.setText(1, str(val))
             # item.setFlags(Qt.ItemIsSelectable|Qt.ItemIsEnabled)
-            #  TODO: Decide if None types must be displayed
             item.setIcon(
                 0, FIcon(0xF70A, InfoVariantWidget.colors[val.__class__.__name__])
```

Comment:
We may put colors into common file ? Or in a style.py  ?

Reply:
Maybe in style.py yes, i will do that